dotnet/WPF/VisualLayerIntegration/VisualLayerIntegration/BarGraphUtility/BarBrushHelper.cs

```diff
@@ -0,0 +1,141 @@
+﻿//  ---------------------------------------------------------------------------------
+//  Copyright (c) Microsoft Corporation.  All rights reserved.
+// 
+//  The MIT License (MIT)
+// 
+//  Permission is hereby granted, free of charge, to any person obtaining a copy
+//  of this software and associated documentation files (the "Software"), to deal
+//  in the Software without restriction, including without limitation the rights
+//  to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+//  copies of the Software, and to permit persons to whom the Software is
+//  furnished to do so, subject to the following conditions:
+// 
+//  The above copyright notice and this permission notice shall be included in
+//  all copies or substantial portions of the Software.
+// 
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+//  ---------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using Windows.UI;
+using Windows.UI.Composition;
+
+namespace BarGraphUtility
+{
+    // Create brushes to fill the bars. 
+    public class BarBrushHelper
+    {
+        private Compositor compositor;
+        private Random rand;
+
+        public BarBrushHelper(Compositor c)
+        {
+            compositor = c;
+            rand = new Random();
+        }
+
+        internal CompositionBrush[] GenerateSingleColorBrush(int numBrushes, Color color)
```

Comment:
updated

Reply:
Naming would make more sense to me if it was CreateColorBrush - because that would be consistent with the method on Compositor.
The name GenerateRandomColorBrushes(...) makes sense for the other method because it is generating values.

---
In reply to: [266607962](https://github.com/Microsoft/Windows.UI.Composition-Win32-Samples/pull/1#discussion_r266607962) [](ancestors = 266607962)